Update README.md

ineiti opened the pull request:
Add a `Privacy` section with the information sent to ChatGPT

searchableguy commented on README.md:
```diff
@@ -29,6 +29,17 @@ wz open google.com in chrome
 
+## Privcacy
+
+This script will send the following information to ChatGPT:
```
```suggestion
Whiz will send the following information to OpenAI:
```

searchableguy commented on README.md:
```diff
@@ -29,6 +29,17 @@ wz open google.com in chrome
+- The basename of your shell path: `process.env.SHELL?.split("/").pop()`, whatever that is
+- Your input prompt
+
+It will not send anything else about your environment like filenames or file content to ChatGPT.
```
```suggestion
No other information is shared with OpenAI.
```

searchableguy commented on README.md:
```diff
@@ -29,6 +29,17 @@ wz open google.com in chrome
+- Your `process.platform`, one of ‘aix’, ‘android’, ‘darwin’, ‘freebsd’, ‘linux’, ‘openbsd’, ‘sunprocess’, or ‘win32’
+- Your `process.arch`, one of  ‘x32’, ‘x64’, ‘arm’, ‘arm64’, ‘s390’, ‘s390x’, ‘mipsel’, ‘ia32’, ‘mips’, ‘ppc’, or ‘ppc64’
+- The basename of your shell path: `process.env.SHELL?.split("/").pop()`, whatever that is
+- Your input prompt
```
```suggestion
- Your request
```

searchableguy commented:
Thanks for the pull request! This is a great addition 😃

Could you please make it shorter and take into consideration the suggestions I have left?

searchableguy commented:
Do you think the following would be easier to read and understand?

This script sends your OS platform, CPU architecture, shell path, and input prompt to OpenAI, excluding other environment details like filenames or content.

ineiti commented:
Updated it with your suggestion - I'm a technical guy, so I prefer the technical proposition you made :)

I also added a warning about not sending sensitive information.

searchableguy commented:
Thanks for your contribution.
